endpoints/openrtb2/auction_test.go

```diff
@@ -102,6 +102,10 @@ func TestJsonSampleRequests(t *testing.T) {
 			"There are both disabled and non-disabled bidders, we expect a 200",
 			"disabled/good",
 		},
+		{
+			"Requests with first party data context info found in imp[i].ext.prebid.bidder,context",
+			"first-party-data",
+		},
```

Comment:
PR #1450 mistakenly removed `TestFirstPartyDataRequests`. This PR reincorporates both `endpoints/openrtb2/sample-requests/first-party-data/valid-context-allowed-with-ext-bidder.json`, `endpoints/openrtb2/sample-requests/first-party-data/valid-context-allowed-with-prebid-bidder.jsoendpoints/openrtb2/sample-requests/first-party-data/valid-context-allowed-with-prebid-bidder.jsonn` to the test suite and updates it to the new JSON tests format.